external/contracts/snapshot/DelegateRegistry.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
Typically, our pattern to import external contracts is to pull the ABI and bytecode into a JSON in the `external/abi` folder. You can fetch both from Etherscan

Here's an example: https://github.com/SetProtocol/set-protocol-v2-contracts/blob/master/external/abi/uniswap/StakingRewards.json

Feel free to ping me if you have any questions about importing